src/routes/auth.js

```diff
@@ -468,7 +467,48 @@ router.get('/verifyRToken', async (req, res) => {
             msg: 'User authenticated successfully',
         });
     } catch (error) {
-        console.log(error);
+        return res.status(401).json({
+            err: true,
+            msg: 'Unauthorized Client',
+        });
+    }
+});
+
+router.post('/getToken', async (req, res) => {
```

Comment:
Can you please change this name to `/getAuthToken` or something similar? It is kind of confusing >.<